Composer/packages/extensions/visual-designer/src/constants/ElementColors.ts

```diff
@@ -8,7 +8,7 @@ const Colors = {
   Black: '#000000',
   AzureGray: '#3C3C41',
   AzureGray2: '#656565',
-  AzureGray3: '#EBEBEB',
+  AzureGray3: '#D7D7D7',
   Gray80: '#B3B0AD',
   Gray60: '#C8C6C4',
```

Comment:
Is it possible to use the colors from the `@ui-fabric/fluent-theme` package?

Reply:
@a-b-r-o-w-n Yes! I would like to do that. We need designer's help on mapping those colors to our theme, do we need an issue to track this?